pkgs/by-name/ma/marksman/package.nix

```
@@ -28,21 +31,19 @@ buildDotnetModule rec {
 
   nugetDeps = ./deps.json;
 
-  dotnet-sdk = dotnetCorePackages.sdk_8_0_4xx-bin;
-  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_9_0;
 
   postInstall = ''
     install -m 644 -D -t "$out/share/doc/${pname}" LICENSE
   '';
 
   passthru = {
     updateScript = ./update.sh;
```

`nix-update` supports fetching nuget deps since a few months which is the reason a shell script was added, please remove that too.
```suggestion
    updateScript = nix-update-script { };
```